features/support/env.rb

```diff
@@ -3,14 +3,28 @@
 require 'open4'
 require 'rspec'
 
+# rubocop:disable all
```

Comment:
Please don't do this. If you need to disable just what is needed. Also be sure to enable afterword.
See [here](https://github.com/Originate/git-town/blob/00d2c2e1ed03ce46f82c3f7024127d810d9f2187/features/support/commit_helpers.rb#L85) for an example